ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -321,3 +322,43 @@ def install_ocs_ui(self):
         self.install_ocs_operator()
         self.verify_ocs_operator_succeeded()
         self.install_storage_cluster()
+
+    def verify_channel(self, timeout_upgrade=600, channel="4.7"):
+        """
+        Verify Channel
+
+        timeout_install (int): Time in seconds to wait
+        channel (str): ocs y version
+
+        """
+        self.navigate_installed_operators_page()
+        self.do_click(self.dep_loc["ocs_operator_installed"])
+        self.do_click(self.dep_loc["subscription"])
+        sample = TimeoutSampler(
+            timeout=timeout_upgrade,
+            sleep=20,
+            func=self.check_element_text,
+            expected_text=f"stable-{channel}",
+        )
+        if not sample.wait_for_func_status(result=True):
+            logger.error(
+                f"OCS upgrade status is not Succeeded after {timeout_upgrade} seconds"
+            )
+            raise TimeoutExpiredError
+
+    def upgrade_ocs_ui(self, channel=None):
+        """
+        Upgrade OCS version via UI
+
+        channel (str): ocs y version
+        """
+        self.navigate_installed_operators_page()
+        self.do_click(self.dep_loc["ocs_operator_installed"])
+        self.do_click(self.dep_loc["subscription"])
+        ocs_version = get_ocs_version()
+        if ocs_version == "4.6" and channel == "4.7":
+            self.do_click(self.dep_loc["channel_4_6"])
+            self.do_click(self.dep_loc["stable-4.7"])
```

Comment:
@OdedViner does this work only for 4.6 to 4.7 upgrade?

Reply:
I think that those values should be added in config for particular ocs version.
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/ocs_version/ocs-4.7.yaml#L5

We already have channel defined in ocs_csv_channel.

I think that channel_4_6 should be defined in 4.6 config as this is loaded when it's start upgrade and this value can be propagated to this function.